src/Microsoft.ML.EntryPoints/MacroUtils.cs

```diff
@@ -75,21 +75,21 @@ public static EvaluateInputBase GetEvaluatorArgs(TrainerKinds kind, out string e
         public sealed class ArrayIPredictorModelInput
         {
             [Argument(ArgumentType.Required, HelpText = "The models", SortOrder = 1)]
-            public PredictorModel[] Model;
+            public PredictorModel[] Models;
```

Comment:
>Models [](start = 36, length = 6)

Same note on tis file, about entry points. 

Also, the EntryPoint project is not directly user facing.. idk if we still want to do it for consistency.

Reply:
This one does't break the tests, but I do agree we might want to rethink touching entry point arguments right now. The effects might propagate to NimbusML for instance.

@TomFinley what do you think?